packages/bundler-plugin-core/src/index.ts

```diff
@@ -92,6 +92,11 @@ export function sentryUnpluginFactory({
       debug: options.debug,
     });
 
+    // Set the User-Agent that Sentry CLI will use when interacting with Sentry
+    process.env[
```

Comment:
super l, just wondering, is there a reason to not just do:

```
process.env.SENTRY_PIPLELINE = ...
```

Reply:
its just lint rules